cmd/argo/commands/submit.go

```diff
@@ -169,6 +174,13 @@ func submitWorkflowFromResource(ctx context.Context, serviceClient workflowpkg.W
 	tempwf := wfv1.Workflow{}
 
 	validateOptions([]wfv1.Workflow{tempwf}, submitOpts, cliOpts)
+	if cliOpts.scheduledTime != "" {
+		_, err := time.Parse(time.RFC3339, cliOpts.scheduledTime)
+		if err != nil {
+			log.Fatalf("scheduled-time contains invalid time.RFC3339 format. time.RFC3339 should be `2006-01-02T15:04:05Z07:00`.  e.g.: `2006-01-02T15:04:05-07:00`")
```

Comment:
We say the same thing twice here
```suggestion
			log.Fatalf("scheduled-time contains invalid time.RFC3339 format (e.g.: `2006-01-02T15:04:05-07:00`)")
```